supergloo/docs/mesh/tutorials-4-security.md

```diff
@@ -0,0 +1,287 @@
+
+Let's take a look at an example SecurityRule:
+
+```yaml
+apiVersion: supergloo.solo.io/v1
+kind: SecurityRule
+metadata:
+  name: productpage-to-reviews
+  namespace: supergloo-system
+spec:
+  allowedMethods:
+  - GET
+  allowedPaths:
+  - /details/*
+  destinationSelector:
+    upstreamSelector:
+      upstreams:
+      - name: default-reviews-9080
+        namespace: supergloo-system
+  sourceSelector:
+    upstreamSelector:
+      upstreams:
+      - name: default-productpage-9080
+        namespace: supergloo-system
+  targetMesh:
+    name: istio
+    namespace: supergloo-system
+
+```
+
+The above routing rule says to only permit those requests:
+
+* Originating from any pods mapped to the service `productpage` in namespace `default` (port here is irrelevant)
+* Destined to any pods backing the `reviews` service in the `default` namespace.
+* With Method `GET`
+* With Path prefixed by `/default/` (note that `*` can be used as a simple wildcard anywhere in the path string)
+
+> Note: Upstreams will be automatically created in Supergloo's installation namespace by the `discovery` pod, 
+which is why the `namespace` on the [Upstream refs](../v1/github.com/solo-io/solo-kit/api/v1/ref.proto.sk) above says 
+`supergloo-system`. The namespace in which the service represents can be different than that where the Upstream lives.
+For more information, see the [Kubernetes Upstream Spec](https://gloo.solo.io/v1/github.com/solo-io/gloo/projects/gloo/api/v1/plugins/kubernetes/kubernetes.proto.sk/#upstreamspec).
+
+Once a single SecurityRule is created, each one acts as a whilelist for traffic, causing all other traffic to be blacklisted.
+
+### Tutorial
+
+Now we'll demonstrate SecurityRules using the Bookinfo app.
+
+First, ensure you've:
+
+- [installed SuperGloo](../../installation)
+- [installed Istio using supergloo](../install)
+- [Deployed the Bookinfo sample app](../bookinfo)
+
+Now let's open our view of the Product Page UI In our browser with the help of `kubectl port-forward`. Run the following command in another terminal window or the background:
+
+```bash
+kubectl port-forward -n default deployment/productpage-v1 9080
+```
+
+Open your browser to http://localhost:9080/productpage:
+
+![Bookinfo Product Page](../../img/bookinfo-default.png "Bookinfo Product Page")
+
+You'll see the `Book Details` and `Book Reviews` subsections of our landing page are 
+working correctly. The `productpage` service which we are connected to queries
+the `details`, `reviews`, and `ratings` services to populate the HTML elements you see
+on the page.
+
+We'll use some **SecurityRules** to show how we can restrict/enable the `productpage` 
+microservice communication with the other 3 services. 
+
+Let's start by enabling access restrictions with a single SecurityRule. The rule we'll create will enable SuperGloo's policy features without whitelisting any communication.
```

Comment:
Additionally this rule is a bit odd as its source and dest selectors are the same service? Can you shed more color on why we'd do that?

Reply:
the point is to enable RBAC while disabling all communication. the only way to do that currently is by creating a security rule with no whitelist.

the api can be changed but it would require some changes to supergloo